txn_writer.go

```diff
@@ -21,15 +21,43 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLES & DOCS - safe to delete!
+#
+## Dependencies define constraints on how dependent projects should be
```

Comment:
Dependencies define constraints on dependent projects. The bit about Gopkg.lock is a bit jargony and I don't think it adds anything.